frontend/templates/troubleshooting/solution.tsx

```diff
@@ -155,6 +169,14 @@ function LinkCards({
          {products.map((product: SectionProduct, index) => (
             <ProductResource key={index} product={product} />
          ))}
+         {partCollections.map(
+            (partCollection: SectionPartCollection, index) => (
+               <PartCollectionResource
+                  key={index}
```

Comment:
Just checking - this is ok with `<ProductResource key={index} ...>` above since they're different lists, right?

Reply:
Do you mean from a UI standpoint? If so, yes. It looks the same as if there were multiple guides or products linked in the section.

Reply:
Oh, I think you mean the index. Yep. I believe index is scope limited to within the map.

Reply:
[639874c](https://github.com/iFixit/react-commerce/pull/2104/commits/639874cc7f00079d396551590beccc991c0e0280)